pkg/apis/work/v1alpha2/binding_types.go

```diff
@@ -322,6 +322,11 @@ type BindingSnapshot struct {
 	Clusters []TargetCluster `json:"clusters,omitempty"`
 }
 
+// Suspension defines the policy for suspending of propagation and suspension of resource binding itself.
+type Suspension struct {
+	*policyv1alpha1.Suspension `json:",inline"`
```

Comment:
Can you remind me why use a pointer type of `pkg/apis/work/v1alpha2/binding_types.go`?

Reply:
We can seperate it's not set or set a zero value by user.

Reply:
One question is, if it is to understand the coupling, why not re-declar all the fields in the binding. What if ScheduleSuspension is also exposed in the PropagationPolicy in the future, according to the current processing?

Reply:
As @RainbowMango mentioned, pp should not expose bindingSuspension now, and we can declare an independent struct to express rb's suspension, this can be done in the future.